scripts/github-status/github-status.rb

```diff
@@ -84,7 +84,7 @@ def print_pr_sha_info(pull)
         print_pr_sha_info(p)
       end
     else
-      puts "#{pull.number}:#{pull.head.sha}" if pull
+      puts "#{pull.number}:#{pull.head.sha}:#{pull.base.ref}" if pull
```

Comment:
There should be a way to trim the jenkins display, but thats unrelated to the change in this place (its in the jenkins job definition)